packages/node_modules/@cerebral/forms/src/rules.js

```diff
@@ -80,7 +80,9 @@ const rules = {
     return value === get(state`${field}.value`)
   },
   maxLength(value, length) {
-    return value.length <= length
+    return (
+      !rules.isExisty(value) || value.length <= length
```

Comment:
I think we could also add
`|| rules.isEmpty(value) `
like in the `regexp` and `minLength` rules.

Reply:
@henri-hulski That is a very good suggestion. Also add null, undefined test to the rules.js test file....

Reply:
`isExisty` actually checks null and undefined.

Reply:
Yes, but we don't know if all rules have isExisty....

Reply:
And if they should...

Reply:
Most of the rules already have isExisty through regexp.
But yeah maybe it should be added to all rules (where it makes sense).

Reply:
Ok, just commented on the undefined null checks in test..... Think we need more of these to catch errors...